runtime/src/main/java/org/corfudb/runtime/clients/SequencerClient.java

```diff
@@ -10,6 +10,8 @@
 
 import lombok.Getter;
 import lombok.Setter;
+
+import org.corfudb.protocols.wireprotocol.CorfuMsg;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.corfudb.protocols.wireprotocol.CorfuMsg'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)